package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "add-variable-declarations",
+  "version": "0.0.0-development",
   "description": "Add variable declarations for global assignments.",
-  "main": "dist/add-variable-declarations.js",
-  "jsnext:main": "dist/add-variable-declarations.mjs",
+  "main": "src/index.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "build": "rollup -c",
     "test": "mocha",
-    "prepublish": "npm run build",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "prepublish": "tsc --project ."
```

Comment:
The "right way" to do this is unclear to me. WebStorm can compile `.ts` to `.js` as you edit, so there's no need to compile separately. In VS Code it seems the recommended way is to run `tsc -w` in a terminal window.